Makefile.am

```diff
@@ -130,27 +130,36 @@ endif
 
 ### Tests (make check)
 
-if ENABLE_DOCS
 TESTS = tests/optionaltest tests/mantest tests/jqtest tests/onigtest tests/shtest tests/utf8test tests/base64test
-else
-TESTS = tests/optionaltest tests/jqtest tests/onigtest tests/shtest tests/utf8test tests/base64test
-endif
 TESTS_ENVIRONMENT = NO_VALGRIND=$(NO_VALGRIND)
 
+# We build the docs from the manpage yml. If no changes have been made to the
+# manpage, then we'll end up using the cached version. Otherwise, we need to
+# rebuild it- making changes to the manpage without having the docs means your
+# tests won't run. If you aren't making changes to the examples, you probably
+# don't care. But if you are, then you need to run the tests anyway.
+check_DATA = tests/man.test
+tests/man.test: $(srcdir)/docs/content/manual/manual.yml
```

Comment:
Do we need a dependency on this target?

Reply:
Yep, this forces `make check` to update `tests/man.test` if the `manual.yml` has been updated. It's intended to prevent changes to the manual not getting reflected in the prebuilt `man.test`.